couchbase/tests/test_unit.py

```diff
@@ -52,3 +59,31 @@ def test__get_query_monitoring_data():
     """
     couchbase = Couchbase('couchbase', {}, {})
     couchbase._get_query_monitoring_data({'query_monitoring_url': 'http://foo/bar'})
+
+
+@pytest.mark.parametrize(
+    'test_case, extra_config, expected_http_kwargs',
+    [
+        (
+            "new auth config",
+            {'username': 'new_foo', 'tsl_verify': True},
+            {'auth': ('new_foo', 'password'), 'verify': True},
+        ),
+        ("legacy config", {'user': 'new_foo', 'ssl_verify': True}, {'auth': ('new_foo', 'password'), 'verify': True}),
```

Comment:
IMO `ssl_verify = false` should be tested too. The default is true, so even if there is a typo for `ssl_verify` e.g. `ssl_verifyy` the test will still pass.

Reply:
there was a typo indeed